src/test/java/selenium/DomainHelper.java

```diff
@@ -3,8 +3,8 @@
 public class DomainHelper {
     
     public static String getBaseUrl() {
-      //return "http://localhost:8080/webapp";
-        return "http://test.elimu.ai";
+      return "http://localhost:8080/webapp";
```

Comment:
@raidenovski This URL will not work on the test server.

Reply:
@jo-elimuai My mistake, will change